doc/source/operating.rst

```diff
@@ -153,7 +153,7 @@ For example, if you want to keep a year of data with a one minute resolution::
 
 Then::
 
-    size in bytes = 525 600 bytes × 6 = 3 159 600 bytes = 3 085 KiB
+    size in bytes = 525 600 bytes × 8 = 4 204 800 bytes = 4 106 KiB
```

Comment:
to jd,
    because one point uses 8 bytes, so 'size in bytes' just compute the total storage for one archive policy.
    if use 6 archive policy, then the total storage shoule be 'size in bytes' * 6. This is mentioned below in the doc.